config.yaml

```diff
@@ -41,7 +41,7 @@ commands:
         short_flag: s
   oncall:
     executable: /home/bundle/cog-command
-    description: "Returns the current oncall for all, or a specified service"
+    description: Returns the current oncall for all, or a specified service
```

Comment:
Maybe `s/Returns/Retrieve/` to stay consistent. Totally not a big deal, just feeling nitpicky.